Mathlib/Init/Data/Int/Basic.lean

```diff
@@ -62,6 +66,10 @@ protected theorem ofNat_add_one_out (n : ℕ) : ↑n + (1 : ℤ) = ↑(succ n) :
 #align int.neg_succ_of_nat_inj_iff Int.negSucc_inj
 #align int.neg_succ_of_nat_eq Int.negSucc_eq
 
+/--
+  Let `a` and `b` be in `ℤ`.
+  If `-a = -b`, then `a = b`.
+-/
```

Comment:
I would personally write this more succinctly as "For integers `a` and `b`, `-a = -b` implies `a = b`."
(This is inherently subjective. You mentioned being newcomer-friendly; I am not totally new to Lean any more. I would, however, that mathlib might not be the place to teach people new to Lean, and the target audience might be different.)

In any case, thanks for documenting this lemma - in general, there is lots of room for more documentation, and extending module docstrings is (imho) scarcely needed in many places.